stable/mlrun/templates/mlrun-api-pvc.yaml

```diff
@@ -5,9 +5,8 @@ metadata:
   name: {{ .Values.api.volumes.storageOverride.persistentVolumeClaim.claimName }}
   labels:
     {{- include "mlrun.api.labels" . | nindent 4 }}
-{{- with .Values.api.persistence.annotations }}
-annotations: {{ toYaml . | nindent 4 }}
-{{- end }}
+  annotations:
+    helm.sh/resource-policy: "keep"
```

Comment:
As per @liranbg's suggestion, leave this as a configurable value in the values file, just populate that value with this annotation.